packages/rke2-calico/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,15 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -3,6 +3,12 @@
+ installation:
+   enabled: true
+   kubernetesProvider: ""
++  calicoNetwork:
++    bgp: Disabled
++    ipPools:
++    - natOutgoing: Enabled
++      encapsulation: VXLAN
++      cidr: 10.42.0.0/16
```

Comment:
Just to be clear, the Calico deployment only supports using its own IPAM, not the Kubernetes IPAM?

Reply:
There is a way for Calico to support Kubernetes IPAM but it does not work with VXLAN and throws the error: `VXLAN is invalid for ipPool.encapsulation with Calico CNI and HostLocal IPAM` (trying to understand why in parallel). When using VXLAN, we must use Calico own IPAM, which is set by the value you see in values.yaml.patch. I was thinking about using here your`.Values.global.clusterCIDR` to be in sync with the cluster-cidr